src/test/java/com/scalyr/integrations/kafka/AddEventsClientTest.java

```diff
@@ -438,6 +447,50 @@ public void testTooLargeAddEventsSkipped() throws Exception {
   }
 
 
+  /**
+   * Verify Add Events Requests that exceed maximum add events payload size before compression are not sent.
+   *
+   * In this scenario we test a request which is larger than 6 MB before compression, but smaller than 6 MB
+   * after compression.
+   */
+  @Test
+  public void testTooLargeAddEventsSkippedDeflateCompressedRequest() throws Exception {
+    final int numEvents = 6;
+    final byte[] largeMsgBytes = new byte[1000000];
+    Arrays.fill(largeMsgBytes, (byte)'a');
+    final String largeMsg = new String(largeMsgBytes);
+    AddEventsResponse addEventsResponse;
+
+    // Setup Mock Server
+    server.enqueue(new MockResponse().setResponseCode(200).setBody(ADD_EVENTS_RESPONSE_SUCCESS));
+
+    // Create addEvents request
+    AddEventsClient addEventsClient = new AddEventsClient(scalyrUrl, API_KEY_VALUE, ADD_EVENTS_TIMEOUT_MS, ADD_EVENTS_RETRY_DELAY_MS, this.deflateCompressor);
```

Comment:
Nit: Typically we don't specify `this` unless there is a local variable with the same name.  Applies to other references of `this` below.  Less clutter in the code.

Reply:
Resolved in 36962e6503704c1aa5f95f869f5bcf77704dc66f.